docs/getting-started/keeping-backstage-updated.md

```diff
@@ -0,0 +1,33 @@
+---
+id: keeping-backstage-updated
+title: Keeping Backstage Updated
+description: How to keep your Backstage App updated
+---
+
+Backstage is always improving, so it's a good idea to stay in sync with the
+latest releases. Backstage is more of a library than an application or service;
+similar to `create-react-app`, the `@backstage/create-app` tool gives you a
+starting point that's meant to be evolved.
+
+## Updating Backstage versions with backstage-cli
+
+The Backstage CLI has a command to bump all `@backstage` packages you're using
+to the latest versions:
+[versions:bump](https://backstage.io/docs/cli/commands#versionsbump).
+
+```bash
+npx backstage-cli versions:bump
+```
+
+The reason for bumping all `@backstage` packages at once is dependency between
+Backstage packages. React Context, for example, may not be referentially equal
+if multiple versions of `@backstage/core` are loaded.
+
+## Following @backstage/create-app changes
+
+Staying up to date with new releases will keep your app up to date, but there
+can also be changes to the `@backstage/create-app` template that may be
+beneficial. For that purpose, any changes made to the template are documented
```

Comment:
```suggestion
Staying up to date with new releases will keep your overall application up to date, but there
can also be changes to the `@backstage/create-app` template. Since your `packages/app`
and `packages/backend` packages are established at `create-app` time and not automatically
touched after that, you may want to apply the corresponding changes to those packages by hand.

For that purpose, any changes made to the template are documented
```

Reply:
Something like that? Explaining the cause and process more.

Reply:
I merged in the suggestion locally and then reworked it a bit, please take a look! ❤️